tests/apps/courses/test_models_category.py

```diff
@@ -145,3 +145,34 @@ def test_models_category_get_blogposts_several_languages(self):
         )
         self.assertEqual(BlogPost.objects.count(), 2)
         self.assertEqual(category.get_blogposts().count(), 1)
+
+    def test_models_category_get_children_categories(self):
+        """
+        It should be possible to retrieve the list of direct children page which
+        are Category extensions and not any other type.
+        """
+        empty_category = CategoryFactory(should_publish=True)
+
+        parent_category = CategoryFactory(should_publish=True)
+        CategoryFactory(
+            page_parent=parent_category.extended_object, should_publish=True
+        )
+        CategoryFactory(
+            page_parent=parent_category.extended_object, should_publish=True
+        )
+
+        BlogPostFactory.create_batch(
```

Comment:
I don't see the point of adding this blogpost for the test. 
What you should add are pages of another type below the parent category page to check that they are not returned by the get_children_categories page?

Reply:
@sampaccoud this to ensure there is nothing else than Category object is returned